insights/parsers/fapolicyd_rules.py

```diff
@@ -0,0 +1,27 @@
+"""
+FapolicydRules - file ``/etc/fapolicyd/rules.d/*.rules``
+========================================================
+"""
+
+from insights import parser
+from insights.core import LogFileOutput
+from insights.specs import Specs
+
+
+@parser(Specs.fapolicyd_rules)
+class FapolicydRules(LogFileOutput):
```

Comment:
Generally this PR looks good to me, but since the base class `LogFileOutput` is designed for log-like files, could you please add corresponding explanation in the doc for this parser about why using the `LogFileOutput` as base?

Reply:
@huali027 I can understand the reason why `LogFileOutput` is used to make the scanning easier However it will make more sense to use the generic `Parser` with `parse_content` to collect lines. But for now this is ok.